[2018-08-29T21:53:36.168] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /ccc
request client ip:  ::ffff:192.168.2.228
request body: 
{}
response time: 88
err name: Error
err message: ER_BAD_DB_ERROR: Unknown database 'dome'
err stack: Error: ER_BAD_DB_ERROR: Unknown database 'dome'
    at Handshake.Sequence._packetToError (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Handshake.js:124:18)
    at Protocol._parsePacket (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (D:\koa\koa\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onread (net.js:635:20)
    --------------------
    at Protocol._enqueue (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (D:\koa\koa\node_modules\mysql\lib\Connection.js:118:18)
    at Pool.getConnection (D:\koa\koa\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (D:\koa\koa\utils\myspl_pool.js:37:14)
    at new Promise (<anonymous>)
    at Object.query (D:\koa\koa\utils\myspl_pool.js:36:12)
    at router.post (D:\koa\koa\routes\index.js:37:33)
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\koa\koa\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\koa\koa\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\koa\koa\node_modules\koa-compose\index.js:42:32)
    at app.use (D:\koa\koa\app.js:51:9)
*************** error log end ***************

[2018-08-29T21:54:03.250] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /ccc
request client ip:  ::ffff:192.168.2.228
request body: 
{}
response time: 6
err name: Error
err message: ER_BAD_DB_ERROR: Unknown database 'dome'
err stack: Error: ER_BAD_DB_ERROR: Unknown database 'dome'
    at Handshake.Sequence._packetToError (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Handshake.js:124:18)
    at Protocol._parsePacket (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (D:\koa\koa\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onread (net.js:635:20)
    --------------------
    at Protocol._enqueue (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (D:\koa\koa\node_modules\mysql\lib\Connection.js:118:18)
    at Pool.getConnection (D:\koa\koa\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (D:\koa\koa\utils\myspl_pool.js:37:14)
    at new Promise (<anonymous>)
    at Object.query (D:\koa\koa\utils\myspl_pool.js:36:12)
    at router.post (D:\koa\koa\routes\index.js:37:33)
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\koa\koa\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\koa\koa\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\koa\koa\node_modules\koa-compose\index.js:42:32)
    at app.use (D:\koa\koa\app.js:51:9)
*************** error log end ***************

[2018-08-29T21:54:30.144] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /add
request client ip:  ::ffff:192.168.2.228
request body: 
{}
response time: 5
err name: Error
err message: ER_BAD_DB_ERROR: Unknown database 'dome'
err stack: Error: ER_BAD_DB_ERROR: Unknown database 'dome'
    at Handshake.Sequence._packetToError (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Handshake.js:124:18)
    at Protocol._parsePacket (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (D:\koa\koa\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onread (net.js:635:20)
    --------------------
    at Protocol._enqueue (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (D:\koa\koa\node_modules\mysql\lib\Connection.js:118:18)
    at Pool.getConnection (D:\koa\koa\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (D:\koa\koa\utils\myspl_pool.js:37:14)
    at new Promise (<anonymous>)
    at Object.query (D:\koa\koa\utils\myspl_pool.js:36:12)
    at router.post (D:\koa\koa\routes\index.js:47:33)
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\koa\koa\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\koa\koa\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\koa\koa\node_modules\koa-compose\index.js:42:32)
    at app.use (D:\koa\koa\app.js:51:9)
*************** error log end ***************

[2018-08-29T21:57:09.927] [ERROR] errorLogger - 
*************** error log start ***************
request method: POST
request originalUrl:  /ccc
request client ip:  ::ffff:192.168.2.228
request body: 
{}
response time: 42
err name: Error
err message: ER_BAD_DB_ERROR: Unknown database 'dome'
err stack: Error: ER_BAD_DB_ERROR: Unknown database 'dome'
    at Handshake.Sequence._packetToError (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Sequence.js:47:14)
    at Handshake.ErrorPacket (D:\koa\koa\node_modules\mysql\lib\protocol\sequences\Handshake.js:124:18)
    at Protocol._parsePacket (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:278:23)
    at Parser.write (D:\koa\koa\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:91:28)
    at Socket.<anonymous> (D:\koa\koa\node_modules\mysql\lib\Connection.js:502:10)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onread (net.js:635:20)
    --------------------
    at Protocol._enqueue (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:144:48)
    at Protocol.handshake (D:\koa\koa\node_modules\mysql\lib\protocol\Protocol.js:51:23)
    at PoolConnection.connect (D:\koa\koa\node_modules\mysql\lib\Connection.js:118:18)
    at Pool.getConnection (D:\koa\koa\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (D:\koa\koa\utils\myspl_pool.js:37:14)
    at new Promise (<anonymous>)
    at Object.query (D:\koa\koa\utils\myspl_pool.js:36:12)
    at router.post (D:\koa\koa\routes\index.js:37:33)
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at next (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:45:18)
    at D:\koa\koa\node_modules\koa-router\lib\router.js:346:16
    at dispatch (D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:44:32)
    at D:\koa\koa\node_modules\koa-router\node_modules\koa-compose\index.js:36:12
    at dispatch (D:\koa\koa\node_modules\koa-router\lib\router.js:351:31)
    at dispatch (D:\koa\koa\node_modules\koa-compose\index.js:42:32)
    at app.use (D:\koa\koa\app.js:51:9)
*************** error log end ***************

